### ISSUE TITLE
Revising OMIP frequency request

### ISSUE THREAD

**Issue by durack1** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#398]
@martinjuckes as noted in https://github.com/WCRP-CMIP/CMIP6_CVs/issues/704#issuecomment-494901155 we'd like to amend the OMIP data request.

The `omip1`, `omip1-spunup`, `omip2`, `omip2-spunup` experiments will all be run with varying model configurations that depending on the model may require 2 to 6 cycles of the 61 (1958 to 2018, JRA55-do) or 62 (1948 to 2009, CORE-2) years of forcing data, before a final cycle is simulated and saved at full monthly (using the `Omon` table output) resolution. For these preceding "spinup" cycles, we want to request data is saved at the annual `Oyr` resolution, to reduce the storage footprint, whilst maintaining output that will be used to diagnose model drift, in addition to AMOC/ACC transports as an integrated diagnostic.

What additional information is required to get such a change implemented for the upcoming `01.00.31` data request release?

ping @stephengriffies @HiroyukiTsujino @jamesorr @taylor13 if the details noted below are correct, we can revise and update the experiment "description" in addition to the brief "experiment" summary in the [CMIP6_CVs](https://github.com/WCRP-CMIP/CMIP6_CVs), which is displayed at [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html)

**Comment by martinjuckes** [comment]
Hello @durack1 : we need to be clear about what changes are needed here, and whether it is worth the confusion which would be caused by introducing new variables (changing from monthly to annual frequency makes a new variable ... a simple change, but it still has the potential to be disruptive, which is why we have promised modeling groups that we will avoid such changes if possible). 

The Omon table has 292 variables. 80 of these are 3d fields on model levels, for the remaining 212 I'm not sure that there is justification for making a change: the data volumes for the monthly surface fields are not generally considered to be a problem. Judging from what you have written above, I think we should restrict the changes to the 3d fields.

Of the 80 3d fields, 58 are already requested in Oyr, so one option would be to simply remove the monthly form of these 58 variables from the request.

There are 22 variables which are requested as 3d fields in Omon and not yet requested in Oyr:
```agessc: Sea Water Age Since Surface Contact
bigthetao: Sea Water Conservative Temperature
ficeberg: Water Flux into Sea Water from Icebergs
hfibthermds: Heat Flux into Sea Water Due to Iceberg Thermodynamics
hfrunoffds: Temperature Flux Due to Runoff Expressed as Heat Flux into Sea Water
hfsifrazil: Heat Flux into Sea Water Due to Frazil Ice Formation
hfsnthermds: Heat Flux into Sea Water Due to Snow Thermodynamics
masscello: Ocean Grid-Cell Mass per Area
obvfsq: Square of Brunt Vaisala Frequency in Sea Water
rsdo: Downwelling Shortwave Radiation in Sea Water
so: Sea Water Salinity
thetao: Sea Water Potential Temperature
thkcello: Ocean Model Cell Thickness
umo: Ocean Mass X Transport
uo: Sea Water X Velocity
vmo: Ocean Mass Y Transport
vo: Sea Water Y Velocity
wmo: Upward Ocean Mass Transport
wo: Sea Water Vertical Velocity
zfullo: Depth Below Geoid of Ocean Layer
zhalfo: Depth Below Geoid of Interfaces Between Ocean Layers
```
These could be added if @taylor13 accepts that the disruption of adding new variables is justified.

At the moment the request is for 310 years of data from both omip1 and omip1-spinup a 295 years from omip2 and omip2-spinup (which is the duration recorded in the experiment CV). How does this relate to what you are describing above? Do you want the Omon output to be supplied for only the last 62 years of all 4 experiments?

**Comment by matthew-mizielinski** [comment]
> Of the 80 3d fields, 58 are already requested in Oyr, so one option would be to simply remove the monthly form of these 58 variables from the request.

If this has any impact outside of OMIP, could I plead for not removing any monthly variables at this point in time. It will get too confusing if we process one ensemble member at data request version 01.00.29 and then we choose to process another at 01.00.32 (unlikely but possible) at a later date if variables such as Omon/thetao disappear.

**Comment by durack1** [comment]
@matthew-mizielinski I agree, any changes must be quarantined to the OMIP experiments only (`omip1`, `omip1-spunup`, `omip2`, and `omip2-spunup`) and leave the DECK/CMIP and other experiments intact
